lmp/jobserv.yml

```diff
@@ -1,6 +1,6 @@
 timeout: 540  # a build with no cache is quite slow
 triggers:
-  - name: build-release
+  - name: build-kirkstone
```

Comment:
The machines for kirkstone were actually aligned with build-release-stable, build-release is actually aligned with scarthgap.

Reply:
This is what we should have done at the time of creating the branch as I mentioned in the previous comment.
_When we created the kirkstone branch we should have changed the configuration of the build-release job that follows refs/heads/main to start following the refs/heads/kirkstone._

Reply:
the reason of this remaining is because we will have more than one type of build-release, for example v94.x kirkstone based and v95+ scarthgap based.